src/docfx/build/xref/InternalXrefMapBuilder.cs

```diff
@@ -147,7 +147,7 @@ private InternalXrefSpec[] AggregateXrefSpecs(string uid, InternalXrefSpec[] spe
             // uid conflicts with overlapping monikers
             // log an warning and take the first one order by the declaring file
             var conflictsWithMoniker = specsWithSameUid.Where(x => x.Monikers.Count > 0).ToArray();
-            if (CheckOverlappingMonikers(specsWithSameUid, out var overlappingMonikers))
+            if (CheckOverlappingMonikers(specsWithSameUid, _errors, out var overlappingMonikers))
```

Comment:
no need to pass in `_errors` since it can be accessed in the method

Reply:
fixed.